src/components/structures/FontSlider.js

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2020 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class Slider extends React.Component {
+    static propTypes = {
+
+        // A callback for the new value onclick
+        updateFontSize: PropTypes.func,
```

Comment:
we can make this a bit more generic of a component by removing references to 'font' throughout the file. For example, naming this `onSelectionChange`, renaming the file to `Slider.js`, and updating the CSS classes to match.

Also this feels like it should be in `elements` instead of `structures` as it's not doing a lot of major work.

Reply:
Done